packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -235,10 +253,17 @@ export default class EthereumAnchorService extends AnchorService {
             throw new Error('Invalid chain ID according to CAIP-2')
```

Comment:
Can we update this error message (it's not exactly accurate for example if I were to pass `bip122:000000000019d6689c085ae165831e93`)


Maybe just `Unsupported chain ID. Must be eip155 namespace.`

Reply:
Done